src/index.js

```diff
@@ -1,92 +1,90 @@
+  const catHugImg = document.getElementById('catHugImg');
+  const catHealthImg = document.getElementById('catHealthImg');
+  const catDietImg = document.getElementById('catDietImg');
+  const catGalleryImg = document.getElementById('catGalleryImg');
+  const catImg = document.getElementById('catImg');
+  const catContainer = document.getElementById('catContainer');
+
+  // Function to update an element’s properties
+  const updateElement = (element, { text = null, color = null, image = null, html = null } = {}) => {
+    if (text !== null) element.textContent = text;
+    if (color !== null) element.style.color = color;
+    if (image !== null) element.src = image;
+    if (html !== null) element.innerHTML = html;
+  };
+
+  // Function to handle hover effects on elements
+  const handleHover = (element, { hoverText = null, leaveText = null, hoverColor = null, leaveColor = null, hoverImage = null, leaveImage = null } = {}) => {
+    element.addEventListener('mouseenter', () => {
+      updateElement(element, { text: hoverText, color: hoverColor, image: hoverImage });
+    });
+    element.addEventListener('mouseleave', () => {
+      updateElement(element, { text: leaveText, color: leaveColor, image: leaveImage });
+    });
+  };
+
+  // Set hover effects on various elements with the `handleHover` function
+  handleHover(catBubble, { hoverText: 'Meowww', leaveText: 'Pet me' });
+  handleHover(catNameLogo, { hoverText: 'MY BABY', leaveText: 'SOLOMON', hoverColor: '#6ED5DB', leaveColor: 'black' });
+  handleHover(catHugImg, { hoverImage: 'images/index_images/cat_hug.gif', leaveImage: 'images/index_images/cat_hug.png' });
+  handleHover(catHealthImg, { hoverImage: 'images/index_images/cat_health.gif', leaveImage: 'images/index_images/cat_health.png' });
+  handleHover(catDietImg, { hoverImage: 'images/index_images/cat_diet.gif', leaveImage: 'images/index_images/cat_diet.png' });
+  handleHover(catGalleryImg, { hoverImage: 'images/index_images/cat_gallery.gif', leaveImage: 'images/index_images/cat_gallery.png' });
+
+  // Function to create a message at the center of an element or return an existing one
+  function createOrGetMessage(elem, html) {
+    let existingMessage = document.getElementById('center-message');
+
+    if (!existingMessage) {
+      let message = document.createElement('div');
+      message.id = 'center-message';
+
+      message.style.cssText = `
+                font-size: 36px;
+                margin: 20px 0 10px;
+                background: linear-gradient(135deg, #6ED5DB, #67A6E8);
+                color: white;
+                border: none;
+                border-radius: 12px;
+                cursor: pointer;
+                box-shadow: 0 4px 15px rgba(0, 0, 0, 0.2);
+                transition: all 0.3s ease;
+                position: absolute;  
+                padding: 5px; 
+                z-index: 10000; 
+                text-align: center;
+            `;
```

Comment:
Usually it's more readable when you put CSS to CSS files and apply certain class to the element in js manipulations